x-pack/filebeat/module/coredns/log/test/coredns-json.log-expected.json

```diff
@@ -0,0 +1,101 @@
+[
+    {
+        "@timestamp": "2019-02-12T00:27:28.903Z",
+        "coredns.dnssec_ok": false,
+        "coredns.id": "21583",
+        "coredns.query.class": "IN",
+        "coredns.query.name": "httpbin.org.cluster.local.",
+        "coredns.query.size": 43,
+        "coredns.query.type": "A",
+        "coredns.response.code": "NXDOMAIN",
+        "coredns.response.flags": [
+            "QR",
+            "RD",
+            "RA"
+        ],
+        "coredns.response.size": 136,
+        "ecs.version": "1.0.0",
+        "event.dataset": "coredns.log",
+        "event.duration": 102078,
+        "event.module": "coredns",
+        "fileset.name": "log",
+        "input.type": "log",
+        "log.level": "INFO",
+        "log.offset": 0,
+        "message": "{\"message\":\"2019-02-12T00:27:28.903Z [INFO] 172.17.0.4:36413 - 21583 \\\"A IN httpbin.org.cluster.local. udp 43 false 512\\\" NXDOMAIN qr,rd,ra 136 0.000102078s\", \"stream\": \"stdout\", \"time\": \"2019-02-12T00:27:28.903433597Z\", \"kubernetes\": { \"container\": { \"name\": \"coredns\" }, \"node\": { \"name\": \"minikube\" }, \"pod\": { \"uid\": \"d57d545e-2a9d-11e9-995f-08002730e0dc\", \"name\": \"coredns-86c58d9df4-jwhsg\" }, \"namespace\": \"kube-system\", \"replicaset\": { \"name\": \"coredns-86c58d9df4\" }, \"labels\": { \"pod-template-hash\": \"86c58d9df4\", \"k8s-app\": \"kube-dns\" } } }",
```

Comment:
Is this really the expected output that we still have here all json as encoded message?

Reply:
I will make it log.original - the same as the other one

Reply:
Well, the test script does not like log.original. But according to ECS, the original log should go to log.original.

I have stripped the text to only coredns part, and leave it as "message" for now.